DataProc/MergeOutputs/mergeByComponent.C

```diff
@@ -181,6 +179,12 @@ void CopyCPass(const char* alienFileList, const char* outputFileList, Int_t time
     TString src("");
     src.ReadLine(inputFile);
     if (src.IsNull()) continue;
+    if (!counter && !gGrid) {
```

Comment:
Conditionally connect to the Grid. Grid input files are always `alien://...` so it should work there.